packages/@react-aria/autocomplete/docs/useAutocomplete.mdx

```diff
@@ -47,6 +48,8 @@ Autocomplete can be implemented using the [&lt;datalist&gt;](https://developer.m
 
 ## Anatomy
 
+<Anatomy />
```

Comment:
I don't think this is the right anatomy. This is for `useSearchAutocomplete` which is a different thing from `useAutocomplete`.

Reply:
ah, might've been copied in from those hooks docs